README.md

```diff
@@ -11,6 +11,10 @@ Here's the rundown:
  * 💻 Built for Developers: The engineers building software are the best equipped to fix bugs, including security bugs. StackHawk does security, but is built for engineers like you.
  * 🤖 Simple to Automate in CI: Application security tests belong in CI, running tests on every PR. Adding StackHawk tests to a DevOps pipeline is easy.
 
+## Getting Started
+ * Follow directions in our [quickstart guide](QUICKSTART.md)
```

Comment:
I think this should link out to our HawkDocs quick start guide, and improve that doc if necessary.
https://docs.stackhawk.com/hawkscan/#quickstart

To cover the topic of quickly getting started with HawkScan in GitHub Actions, we should also link out to our GitHub Actions integration doc in HawkDocs, and improve that if necessary.
https://docs.stackhawk.com/continuous-integration/github-actions.html